### PR TITLE
Added locale to pagination request of data-navigation

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | - 
| Related issues/PRs | -
| License | MIT
| Documentation PR | -

#### What's in this PR?

The locale-parameter was not sent with the pagination request, although the API demanded it. This PR fixes this issue.

### CODE REVIEW

File: husky_components/data-navigation/main.js [hunk 1]
```diff
@@ -338,17 +338,18 @@ define([
...
-                    .then(function(data) {
-                        var children = data._embedded[this.options.resultKey] || [];
-                        this.data.children = this.data.children.concat(children);
-                        this.data.hasNextPage = this.page < data.pages;
-                        this.currentView.append(children, this.options);
-
-                        this.hideLoader();
-                        def.resolve();
-                    }.bind(this));
+                this.page += 1;
```
Comment (danrot) [inline husky_components/data-navigation/main.js]: Why this change? And would prefer `++this.page`

Comment (marcelmoosbrugger) [inline husky_components/data-navigation/main.js]: It's just a matter of taste. I'm not a fan of "++" and "--" as they have side-effects. In my opinion "+=" makes makes it more clear that there's an assignment happening.
